libse/TimeCode.cs

```diff
@@ -58,7 +58,7 @@ public static double ParseHHMMSSFFToMilliseconds(string text)
             return 0;
         }
 
-        public TimeCode() : this(0)

```

Comment:
there is no need for calling this(0) `_totalmilliseconds` will be initialized to zero when `new TimeCode()`